README.md

```diff
@@ -9,76 +9,92 @@ SAML: https://wiki.oasis-open.org/security/FrontPage
+
+The `samlconf` script takes multiple parameters:
+
+```
+NAME
+       samlconf - Runs the SAML Conformance Tests against an IdP or an SP
+
+SYNOPSIS
+       samlconf [-i path] [--idpMetadata path]
+           [-p path] [--plugins path] 
+
+DESCRIPTION
+       Runs the SAML Conformance Tests which tests the complance of an IdP and/or an SP
+       with the SAML Specifications. If a compliance issue is identified, a 
+       SAMLConformaceException will be throw with an explanation of the error an a direct
+       quote from the specification. All of the parameters are optional and if they are 
+       not provided, the default values will use DDF's parameters. All parameters must 
+       be given one times.
+
+OPTIONS
+       -i path
+           The path to the IdP metadata. If it is not given, the default IdP metadata
+           is /conf/idp-metadata.xml.
+           
+       --idpMetadata path
+           The path to the IdP metadata. If it is not given, the default IdP metadata
+           is /conf/idp-metadata.xml.
+           
+       -p path
+            The path to the pluggable portion. If it is not given, the default plugin
+            is /plugins/ddf-plugins-1.0-SNAPSHOT.jar.
+    
+       --plugins path
+            The path to the pluggable portion. If it is not given, the default plugin
+            is /plugins/ddf-plugins-1.0-SNAPSHOT.jar.
+```
+
+
+> NOTE
+> 
+> In order for the test kit to execute properly, you must configure both the test kit's and your IdP's/SP's metadata, as well as implement plugins
 for the user-handled portions of SAML profiles. See "Metadata" and "Plugins" for instructions.
 
 ### Metadata
 * If testing an IdP:
-  * **TODO** *This requires the user to rebuild and is not an optimal
-  configuration solution. A solution that works with a pre-built artifact
-  should be found.* Configure the test kit with your IdP's metadata by updating
-  `<PATH_TO_PROJECT>/distribution/command-line/src/main/resources/idp-metadata.xml`
-  prior to building.
-  * Configure your IdP with the test kit's metadata from
-  `<PATH_TO_PROJECT>/distribution/command-line/src/main/resources/test-sp-metadata.xml`.
+  * Provide your IdP's metadata file path to the `samlconf` script using `-i` or `--idpMetadata`.
+  * Configure your IdP with the test kit's SP metadata from
+  `<PATH_TO_PROJECT>/distribution/command-line/src/main/resources/test-sp-metadata.xml`
+  or `samlconf-1.0-SNAPSHOT/conf/test-sp-metadata.xml` from the distribution.
   
 * **TODO** If testing an SP:
-  * Place your SP's metadata into `sp-metadata.xml` under `<PATH_TO_PROJECT>/distribution/command-line/src/main/resources`.
-  * Copy `test-idp-metadata.xml` under `<PATH_TO_PROJECT>/distribution/command-line/src/main/resources` into your SP.
+  * Provide your SP's metadata file path to the `samlconf` script using `-s` or `--spMetadata`.
```

Comment:
❓ Is this still valid? This parameter is not included in the man page-like section above.

Reply:
It doesn't exist yet but that will be what we do. I will add a note that it's not implemented yet on the README.